test/unit/src/audio/FftUnit.cpp

```diff
@@ -3,7 +3,7 @@
 // FIXME: OOURA roundtrip FFT seems to be broken on windows for sizeFft = 4 (https://github.com/cinder/Cinder/issues/1263)
 #if ! defined( CINDER_MSW )
 
-#include "catch.hpp"
+#include "../catch.hpp"

```

Comment:
I'm not familiar enough with CMAKE to say how to set the search paths or for it to be clear what you mean by build the subdirs.

The catch unit tests can all be (and should be, I believe) compiled into a single binary that can select different test cases to run. So I visualize the file hierarchy as a bunch of things relative to the root of the test project. Effectively, you're building one thing from the root of the `src/` directory and linking in things from the subdirectories. So in XCode, there's just one project at the root and all the include directories are handled therer.